homeassistant/components/xiaomi_miio/button.py

```diff
@@ -19,22 +22,33 @@
     KEY_DEVICE,
     MODEL_AIRFRESH_A1,
     MODEL_AIRFRESH_T2017,
+    MODELS_VACUUM,
 )
 from .device import XiaomiCoordinatedMiioEntity
 
+# Fans
 ATTR_RESET_DUST_FILTER = "reset_dust_filter"
 ATTR_RESET_UPPER_FILTER = "reset_upper_filter"
 
+# Vacuums
+METHOD_VACUUM_RESET_CONSUMABLE = "consumable_reset"
+ATTR_RESET_VACUUM_MAIN_BRUSH = "reset_vacuum_main_brush"
+ATTR_RESET_VACUUM_SIDE_BRUSH = "reset_vacuum_side_brush"
+ATTR_RESET_VACUUM_FILTER = "reset_vacuum_filter"
+ATTR_RESET_VACUUM_SENSOR_DIRTY = "reset_vacuum_sensor_dirty"
+
 
 @dataclass
 class XiaomiMiioButtonDescription(ButtonEntityDescription):
     """A class that describes button entities."""
 
     method_press: str = ""
+    method_press_params: Any = None
```

Comment:
Shouldn't this be 
```suggestion
    method_press_params: Consumable | None = None
```

Reply:
I used `Any` here since there are other types of params for other commands (or devices), should I limit it to `Consumable` now?

Reply:
> I used `Any` here since there are other types of params for other commands (or devices), should I limit it to `Consumable` now?

@frenck What do you think?

Reply:
> I used Any here since there are other types of params for other commands (or devices), should I limit it to Consumable now?

@zry98, right now there are only 2 scenarios: `method_press_params=None` for previous `XiaomiMiioButtonDescription` items, and the new ones from this PR, which use `Consumable`, so the suggested typing by Franck is the good one: `method_press_params: Consumable | None = None` 👍

(you can just press the 'Commit suggestion' button to accept it 😜)